windows-msi/script/ACL.js

```diff
@@ -171,7 +171,8 @@ function SetACL() {
     targetDir = TrimTrailingSlash(targetDir);
 
     try {
-        RunIcacls('\"' + targetDir + '\" /inheritance:r /grant "Administrators:(OI)(CI)F" /grant "System:(OI)(CI)F" /grant "Users:(OI)(CI)RX"');
+        // Administrators, SYSTEM, Users
+        RunIcacls('\"' + targetDir + '\" /inheritance:r /grant "*S-1-5-32-544:(OI)(CI)F" /grant "*S-1-5-18:(OI)(CI)F" /grant "*S-1-5-32-545:(OI)(CI)RX"');
```

Comment:
While this looks good, we are only removing inherited permissions and setting new explicit ones. Any explicit permission that was already present will remain unchanged. Is this what we want? Or should we be resetting acls first and then setting this? 

For example: with 2.6.8 installed in to a folder with `foo\selva:(OCI)(CI)F` in addition to inherited acls,
on upgrading to 2.6.11 the above explicit permission will be retained and new explicit one's for SYSTEM, Users and Administrators added.

Reply:
This specific commit doesn't alter the logic. It just has slipped from my mind that build-in groups might have localized names (and it is not the case for my Finnish localization by some reasons).

One can argue that if the user has explicitly assigned a permission to the installation directory he knows what he is doing. On the other hand indeed it could weaken the security.

Note that the impact of this code is somewhat limited, since starting from 2.6.9 users are not able to customize the installation directory.

Reply:
Yes, I was asking something I should have asked in response to the original commit in January -- did not occur to me that time :)

As for the fix here to use SIDs to avoid the issue of localization of group names, looks good to me though I do not have a localized system to test this on.